x/xion/keeper/msg_server.go

```diff
@@ -124,7 +125,7 @@ func (k msgServer) MultiSend(goCtx context.Context, msg *types.MsgMultiSend) (*t
 		outputs = append(outputs, banktypes.NewOutput(feeCollectorAcc, totalPlatformCoins))
 	}
 
-	err := k.bankKeeper.InputOutputCoins(ctx, msg.Inputs, outputs)
+	err := k.bankKeeper.InputOutputCoins(ctx, msg.Inputs[0], outputs) // TODO decide on how to handle multiple inputs
```

Comment:
Interesting. This is a new feature I assume? We intend for this handler to mimic the bank handler, except withholding a specific %. I suppose we could withhold the amount from each input